File: C1.UWP.FlexGrid/CS/FlexGridSamples/Data/NorthwindData.cs
using System;
using System.IO;
using System.Reflection;
using System.Collections.Generic;
using System.Xml.Serialization;
using Windows.Storage;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace FlexGridSamples
{
    public class NorthwindData
    {
        [Display(Name = "CustomerID")]
        public string CustomerID { get; set; }

        [Display(Name = "CompanyName")]
        public string CompanyName { get; set; }

        [Display(Name = "ContactName")]
        public string ContactName { get; set; }

        [Display(Name = "ContactTitle")]
        public string ContactTitle { get; set; }

        [Display(Name = "Address")]
        public string Address { get; set; }

        [Display(Name = "City")]
        public string City { get; set; }

        [Display(Name = "PostalCode")]
        public string PostalCode { get; set; }

        [Display(Name = "Country")]
        public string Country { get; set; }

        [Display(Name = "Phone")]
        public string Phone { get; set; }

        [Display(Name = "Fax")]
        public string Fax { get; set; }
    }

    public class NorthwindStorage
    {
        public static async Task<List<NorthwindData>> Load()
        {
            try
            {
                Uri resourceUri;
                if (typeof(NorthwindStorage).GetTypeInfo().Module.Name.EndsWith("exe") ||
                    Windows.ApplicationModel.DesignMode.DesignModeEnabled)
                {
                    resourceUri = new Uri("ms-appx:///Resources/Northwind.xml");
                }
                else
                {
                    resourceUri = new Uri("ms-appx:///FlexGridSamplesLib/Resources/Northwind.xml");
                }
                var file = await StorageFile.GetFileFromApplicationUriAsync(resourceUri);
                var fileStream = await file.OpenAsync(FileAccessMode.Read);
                var xmls = new XmlSerializer(typeof(List<NorthwindData>));
                return (List<NorthwindData>)xmls.Deserialize(fileStream.AsStream());
            }
            catch(Exception e)
            {
                throw new Exception(Strings.FileNotFoundException);
            }
        }
    }
}

File: C1.UWP.FlexGrid/VB/FlexGridSamples/Data/NorthwindData.vb
Imports System.ComponentModel.DataAnnotations
Imports System.Threading.Tasks
Imports Windows.Storage
Imports System.Xml.Serialization
Imports System.Collections.Generic
Imports System.Reflection
Imports System.IO
Imports System
Imports Windows.Storage.Streams

Public Class NorthwindData
    <Display(Name:="CustomerID")>
    Public Property CustomerID() As String

    <Display(Name:="CompanyName")>
    Public Property CompanyName() As String

    <Display(Name:="ContactName")>
    Public Property ContactName() As String

    <Display(Name:="ContactTitle")>
    Public Property ContactTitle() As String

    <Display(Name:="Address")>
    Public Property Address() As String

    <Display(Name:="City")>
    Public Property City() As String

    <Display(Name:="PostalCode")>
    Public Property PostalCode() As String

    <Display(Name:="Country")>
    Public Property Country() As String

    <Display(Name:="Phone")>
    Public Property Phone() As String

    <Display(Name:="Fax")>
    Public Property Fax() As String
End Class

Public Class NorthwindStorage
    Public Shared Async Function Load() As Task(Of List(Of NorthwindData))
        Try
            Dim resourceUri As Uri
            If GetType(NorthwindStorage).GetTypeInfo().[Module].Name.EndsWith("exe") OrElse Windows.ApplicationModel.DesignMode.DesignModeEnabled Then
                resourceUri = New Uri("ms-appx:///Resources/Northwind.xml")
            Else
                resourceUri = New Uri("ms-appx:///FlexGridSamplesLib/Resources/Northwind.xml")
            End If
            Dim file As StorageFile = Await StorageFile.GetFileFromApplicationUriAsync(resourceUri)
            Dim fileStream As IRandomAccessStream = Await file.OpenAsync(FileAccessMode.Read)
            Dim xmls As New XmlSerializer(GetType(List(Of NorthwindData)))
            Return CType(xmls.Deserialize(fileStream.AsStream()), List(Of NorthwindData))
        Catch e As Exception
            Throw New Exception(Strings.FileNotFoundException)
        End Try
    End Function
End Class